src/opensearch_sql_cli/opensearch_connection.py

```diff
@@ -167,14 +176,14 @@ def execute_query(self, query, output_format="jdbc", explain=False, use_console=
         try:
             if self.query_language == "sql":
                 data = self.client.transport.perform_request(
-                    url="/_plugins/_sql/_explain" if explain else "/_plugins/_sql/",
+                    url="/_plugins/_sql/_explain" if explain else "/_plugins/_sql",
```

Comment:
If we don't remove the trailing `/` we get 404s, I'm not sure if we strongly depend on that slash being there in other functionality but the tests still pass.

Reply:
Since it's passed directly to `perform_request`, I'm guessing not